express-app/models/tutor.js

```diff
@@ -0,0 +1,34 @@
+"use strict";
+
+const Miembro = require('./miembro.js');
+
+//options for relations
+const relationOptions = [
+  "madre",
```

Comment:
Are this going to be displayed as options?
In that case it is better to use upper case